soccer/src/soccer/planning/planner/path_target_planner.hpp

```diff
@@ -21,6 +24,12 @@ class PathTargetPlanner : public PlannerForCommandType<PathTargetCommand> {
     Trajectory plan(const PlanRequest& request) override;
     void reset() override { previous_ = Trajectory(); }
 
+    bool is_done() const override;
+
+    // vars to tell if is_done
+    std::optional<LinearMotionInstant> cached_start_instant_;
+    std::optional<LinearMotionInstant> cached_goal_instant_;
```

Comment:
these are public vars, the underscore after the name should just be used on private vars.
I'm assuming though these should just be made private.

Reply:
yes, they should be private, good catch